layers/meta-balena-beaglebone/recipes-bsp/u-boot/u-boot-ti-staging_%.bbappend

```diff
@@ -1,21 +1,18 @@
 UBOOT_KCONFIG_SUPPORT = "1"
 inherit resin-u-boot
 
-UBOOT_MACHINE = "am335x_evm_config"
-
 FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"
+
 SRC_URI:append = " \
-    file://0001-Integrate-with-Balena-environment-configuration.patch \
-    file://0002-am335x_evm_defconfig-Reduce-SPL-.rodata-size.patch \
-    file://0005-Autoboot-keyboard-beaglebone-fixes.patch \
-    file://0001-Add-support-for-BeagleBoard.org-PocketBeagle.patch \
-    file://uEnv.txt_internal \
-    "
+    file://0001-Integrate-with-Balena-u-boot-environment.patch \
+"
 
-SRC_URI:append:beaglebone-pocket = " \
-    file://0001-u-boot-Disable-logging-on-UART0.patch \
-    "
+SRC_URI:append:beaglebone-ai64 = " \
+    file://0001-Use-the-BBONEAI-64-B0-as-default-board.patch \
+    file://0001-Look-for-the-dtb-in-the-ti-directory.patch \
+"
 
-do_deploy:append () {
-    install ${WORKDIR}/uEnv.txt_internal ${DEPLOYDIR}
-}
+# The BB-AI64 creates this file for both archs, remove one.
```

Comment:
@floion so far both extra_uEnv.txt files are created empty, but maybe the one that should be removed is the -k3r5 one

Reply:
@perezmeyer it's no problem if the extra_uEnv.txt file is created empty. That is expected